org.eclipse.xtext.tests/testdata/wizard-expectations/mavenTychoJ11/mavenTychoJ11.parent/mavenTychoJ11/pom.xml

```diff
@@ -36,26 +36,6 @@
 					<cleanupDaemonThreads>false</cleanupDaemonThreads><!-- see https://bugs.eclipse.org/bugs/show_bug.cgi?id=475098#c3 -->
 				</configuration>
 				<dependencies>
-					<dependency>
-						<groupId>org.eclipse.emf</groupId>
-						<artifactId>org.eclipse.emf.mwe2.launch</artifactId>
-						<version>${mwe2Version}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.common.types</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.xtext.generator</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
-					<dependency>
-						<groupId>org.eclipse.xtext</groupId>
-						<artifactId>org.eclipse.xtext.xbase</artifactId>
-						<version>${xtextVersion}</version>
-					</dependency>
 					<dependency>
 						<groupId>org.eclipse.xtext</groupId>
 						<artifactId>xtext-antlr-generator</artifactId>
```

Comment:
Oh... we still have this one :(
So maybe we can't get rid of

```xml
<includePluginDependencies>true</includePluginDependencies>
```

couldn't we remove this one as well and put in the TP a reference to the itemis update site?
After all this won't violate anything and the xtext.generator already has an optional dependency on this bundle anyway (IIRC)

Reply:
I mean something like

```xml
<location includeAllPlatforms="false" includeConfigurePhase="true" includeMode="planner" includeSource="true" type="InstallableUnit">
<unit id="de.itemis.xtext.antlr.feature.feature.group" version="2.1.1.v201405091103"/>
<repository location="https://download.itemis.com/updates/releases/2.1.1/"/>
</location>
```

Reply:
as i have mentioned before i have zero idea on legal impacts. at that time i was zero involved in Xtext

Reply:
I think for the code we generate with the wizard it is ok to have a dependency to xtext-antlr-generator - I'd favor a dependency to maven central though - for qos reasons.

Reply:
in that case we have to leave with `<includePluginDependencies>true</includePluginDependencies>`

I'm wondering why not simply removing also that dependency: the facade will download the bundle when running MWE2... after all, if someone doesn't want the jar to be downloaded by the facade they already know they must add the lines above to their TP (in that case, since xtext.generator has an optional dependency on the antlr generator, Tycho should take it from the TP when running maven exec).

I'm pretty sure I checked that in my previous experiments when dealing with the original problem. If you want I can double-check that, but only if you're willing to remove `<includePluginDependencies>true</includePluginDependencies>` and the additional maven dep on `xtext-antlr-generator`. If you think that's not worthwhile it's fine with me (I always tweak the Maven POM and the TP as the first thing anyway).

Reply:
i still dont trust our (itemis) admins

Reply:
sebastian proposed to look at the new tycho use maven stuff feature and consume the antlr generator this way

Reply:
You mean the fact that a TP can now refer to a Maven artifact? That could a cool solution (or a legal trick ;) O:)

Reply:
yes @szarnekow  do you have a hint which tycho version brought that? and how it looks like

Reply:
https://läubisoft.gmbh/en/articles/using-maven-artifacts-in-pde-rcp-and-tycho-builds/

Reply:
Yes, but please keep in mind that this only works with recent versions of Eclipse, if a user wants to activate the TP from Eclipse (of course, if the TP is used only during the Maven build, using Tycho 2.4.0 will be enough).

Reply:
somehow this does not show up my runtimeXtext workspace from dev env although we use m2e 1.18

Reply:
ahh its a separate feature